aospy/test/test_automate.py

```diff
@@ -351,3 +354,25 @@ def test_permute_aux_specs(self, calc_suite, calcsuite_specs):
         assert len(actual) == len(expected)
         for act in actual:
             assert act in expected
+
+
+@pytest.mark.parametrize(
+    ('var'),
+    [(precip_largescale),
+     (condensation_rain)]
+)
+def test_prune_invalid_time_reductions(var):
+    time_options = ['av', 'std', 'ts', 'reg.av', 'reg.std', 'reg.ts', 'None']
+    spec = {
+        'var': var
+    }
+    for i in range(1, 8):
```

Comment:
To be more explicit, replace `range(1, 8)` with `range(1, len(time_options) + 1)`.

Reply:
You are using the string `'None'`, instead of the built-in `None` object.  I think we want the latter.